docs/source/contributing/jupyter_style.md

```diff
@@ -42,15 +46,22 @@ The first cell of all example notebooks should have a level 1 markdown title (th
 
 The date should correspond to the latest update/execution date, at least roughly (it's not a problem if the date is a few days off due to the review process before merging the PR). This will allow users to see which notebooks have been updated lately and will help the PyMC team make sure no notebook is left outdated for too long.
 
-Tags can be anything, but we ask you to try to use [existing tags](https://github.com/pymc-devs/pymc/wiki/Categories-and-Tags-for-PyMC-Examples) to avoid the tag list from getting too long. Each notebook should
-have a single category indicating the level of the notebook. Choose a category from [existing categories](https://github.com/pymc-devs/pymc/wiki/Categories-and-Tags-for-PyMC-Examples#categories).
+The [MyST target](https://myst-parser.readthedocs.io/en/latest/syntax/syntax.html#targets-and-cross-referencing)
+is important to ease referencing and linking notebooks between them.
```

Comment:
Maybe `linking notebooks between each other` is a better expression than `linking notebooks between them`

Reply:
@martinacantaro I took the liberty of converting your comment into a suggestion to make it easier to update